yt_dlp/extractor/hearthisat.py

```diff
@@ -78,27 +78,23 @@ def _real_extract(self, url):
         thumbnail = self._og_search_thumbnail(webpage)
 
         meta_span = r'<span[^>]+class="%s".*?</i>([^<]+)</span>'
-        view_count = str_to_int(self._search_regex(
-            meta_span % 'plays_count', webpage, 'view count', fatal=False))
-        like_count = str_to_int(self._search_regex(
-            meta_span % 'likes_count', webpage, 'like count', fatal=False))
-        comment_count = str_to_int(self._search_regex(
-            meta_span % 'comment_count', webpage, 'comment count', fatal=False))
-        duration = str_to_int(self._search_regex(
-            r'data-length="(\d+)', webpage, 'duration', fatal=False))
-        timestamp = str_to_int(self._search_regex(
-            r'<span[^>]+class="calctime"[^>]+data-time="(\d+)', webpage, 'timestamp', fatal=False))
+        view_count = str_to_int(re.findall(meta_span % 'plays_count', webpage)[-1])
+        like_count = str_to_int(re.findall(meta_span % 'likes_count', webpage)[-1])
+        comment_count = str_to_int(re.findall(meta_span % 'comment_count', webpage)[-1])
+        duration = str_to_int(re.findall(r'data-length="(\d+)', webpage)[-1])
+        timestamp = str_to_int(re.findall(r'<span[^>]+class="calctime"[^>]+data-time="(\d+)', webpage)[-1])
```

Comment:
this effectively makes all these fatal

Reply:
bcoz of https://github.com/yt-dlp/yt-dlp/issues/740#issuecomment-902851207
I'm marking this PR as draft